gorm/app/controllers/app.go

```diff
@@ -9,6 +9,8 @@ import (
 type App struct {
 	*revel.Controller
 	gorm.GormController
+	// Transactional
```

Comment:
Please update import paths so they use the fully qualified path `github.com/revel/modules/gorm`
Also like I said in the module PR, gorm.Controller should embed *revel.Controller inside of it - then there would be no need to embed the *revel.Controller here